docker-compose.yml

```diff
@@ -38,7 +39,7 @@ services:
     image: minio/minio:latest
     container_name: s3backend
     ports:
-      - "9000:9000"
+      - "9090:9000"
```

Comment:
Why has this changed? If we're changing this to 9090 then we'll need to update readmes, people's secretes for the lambdas etc, can we just leave as is?